.github/ISSUE_TEMPLATE/doc_issue.yaml

```diff
@@ -2,7 +2,7 @@ name: Documentation issue
 description: Report a problem with the Mojo docs
 title: "[Docs]"
 labels:
-  - documentation
```

Comment:
What are these changes for?

Reply:
Hey! so yeah this was a commit that was not intended. I tried reverting it but didn't work out. I will try again.

Reply:
I'm not sure. @JoeLoser do you know how to help with this DCO issue?